index.html

```diff
@@ -81,146 +81,263 @@
       </p>
       <p>
-This specification describes a data model for a digital <a>entity
-profile</a> and a collection of digital <a>entity credentials</a> that
-assert verifiable <a>claims</a> about that <a>entity profile</a>. It also
-describes how to express that data model in JSON, JSON-LD, and WebIDL.
+This specification details a data model for verifiable claims where
+credentials such as employee ID cards, driver's licenses, passports,
+educational achievements, and healthcare information may be expressed
+in a variety of data syntaxes such as JSON and XML.
       </p>
     </section>
 
     <section id='sotd'>
-      <p>Drawn initially from
-      [[IDENTITY-CREDENTIALS]], this
-      document is intended to provide a rough outline of the kind of
-      standardization work that a Verifiable Claims Working Group
-      might do.
+      <p>
+Comments regarding this document are welcome. Please file issues
+directly on <a href="https://github.com/w3c/vc-data-model/issues/">GitHub</a>,
+or send them to
+<a href="mailto:public-vc-comments@w3.org">public-vc-comments@w3.org</a>
+(<a href="mailto:public-vc-comments-request@w3.org?subject=subscribe">subscribe</a>,
+<a href="https://lists.w3.org/Archives/Public/public-vc-comments/">archives</a>).
       </p>
-    <p>Comments regarding this document are welcome. Please
-      or file
-      directly on <a href="https://github.com/w3c/vc-use-cases/issues/">GitHub</a>,
-      or send them to
-      <a href="mailto:public-vc-comments@w3.org">public-vc-comments@w3.org</a> (<a href="mailto:public-vc-comments-request@w3.org?subject=subscribe">subscribe</a>,
-      <a href="https://lists.w3.org/Archives/Public/public-vc-comments/">archives</a>).</p>
     </section>
 
     <section>
       <h2>Introduction</h2>
       <p>
-      We typically use credentials, or claims made by
-      others about our identities, to authenticate ourselves and
-      ultimately gain access to various services. For example, we use
-      email addresses to identify ourselves to online services,
-      driver's licenses to prove that we are capable of operating a
-      motor vehicle, university degrees to prove we are well-trained
-      and knowledgeable, and government-issued passports to travel
-      between countries or to access financial services. It is the
-      goal of this specification to provide an easy, standard way to
-      express such credentials on the Web.
+We typically use claims made by others about us to gain access to
+various online services. For example, we use
+email addresses to identify ourselves to online services,
+driver's licenses to prove that we are capable of operating a
+motor vehicle, university degrees to prove we are well-trained
+and knowledgeable, and government-issued passports to travel
+between countries and access financial services. This specification
+provides a standard way to express these sorts of claims on the Web in
+a way that is automatically verifiable.
```

Comment:
Just a note here that whatever changes we make in the abstract need to be copied here.